examples/deploy-heroku/requirements.txt

```
@@ -0,0 +1,3 @@
+slack_bolt
+Flask>=1.1
```

Recently, Flask v2 was released. I'm sure that Bolt itself should work with v2 but can you verify if this example work with the major version, just in case? If there is no issue with v2, we can set the minimum version to 2 for more clarity.
```suggestion
Flask>=2,<3
```